scripts/code_coverage_report/generate_code_coverage_report/Sources/Utils/ShellUtils.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+public struct Shell {
+  static let shared = Shell()
+  public init() {}
```

Comment:
Nit: the `shared` shell and `init` don't really do anything here, and since we're using a static func it's probably better to use a caseless `enum` and treat it as a namespace:

```
public enum Shell {
```

That way we can remove the `static let shared = Shell()` and `public init()` since there's nothing to initialize there.

Happy to chat more about it if you'd like!

Reply:
Yep it is more reasonable to set it as a caseless `enum`, have updated it.